lib/content-info/index.js

```diff
@@ -0,0 +1,315 @@
+    * SP            = <US-ASCII SP, space (32)>
+    * HT            = <US-ASCII HT, horizontal-tab (9)>
+    * CTL           = <any US-ASCII control character (octets 0 - 31) and DEL (127)>
+    * OCTET         = <any 8-bit sequence of data>
+    *
+    * egHeader: inline; filename=testResponse.json
+    * egHeader: inline; filename="test Response.json"
+    */
+    // eslint-disable-next-line max-len
+    fileNameRegex = /;[ \t]*(?:filename)[ \t]*=[ \t]*("(?:[\x20!\x23-\x5b\x5d-\x7e\x80-\xff]|\\[\x20-\x7e])*"|[!#$%&'*+.0-9A-Z^_`a-z|~-]+)[ \t]*/,
+
+    /**
+    * @private
+    * RegExp for extracting filename* from content-disposition header
+    *
+    * RFC 5987 grammar
+    * parameter     = reg-parameter / ext-parameter
+    * ext-parameter = parmname "*" LWSP "=" LWSP ext-value
+    * parmname      = 1*attr-char
+    * ext-value     = charset  "'" [ language ] "'" value-chars
+                   ; like RFC 2231's <extended-initial-value>
+                   ; (see [RFC2231], Section 7)
+    * charset       = "UTF-8" / "ISO-8859-1" / mime-charset
+    * mime-charset  = 1*mime-charsetc
+    * mime-charsetc = ALPHA / DIGIT
+                   / "!" / "#" / "$" / "%" / "&"
+                   / "+" / "-" / "^" / "_" / "`"
+                   / "{" / "}" / "~"
+                   ; as <mime-charset> in Section 2.3 of [RFC2978]
+                   ; except that the single quote is not included
+                   ; SHOULD be registered in the IANA charset registry
+    * language      = <Language-Tag, defined in [RFC5646], Section 2.1>
+    * value-chars   = *( pct-encoded / attr-char )
+    * pct-encoded   = "%" HEXDIG HEXDIG
+                   ; see [RFC3986], Section 2.1
+    * attr-char     = ALPHA / DIGIT
+                   / "!" / "#" / "$" / "&" / "+" / "-" / "."
+                   / "^" / "_" / "`" / "|" / "~"
+                   ; token except ( "*" / "'" / "%" )
+    *
+    * egHeader: attachment;filename*=utf-8''%E4%BD%A0%E5%A5%BD.txt
+    */
+    // eslint-disable-next-line max-len
+    encodedFileNameRegex = /;[ \t]*(?:filename\*)[ \t]*=[ \t]*([A-Za-z0-9!#$%&+\-^_`{}~]+)'.*'((?:%[0-9A-Fa-f]{2}|[A-Za-z0-9!#$&+.^_`|~-])+)[ \t]*/,
+
+    /**
+    * @private
+    *
+    * RegExp to match quoted-pair in RFC 2616
+    *
+    * quoted-pair = "\" CHAR
+    * CHAR        = <any US-ASCII character (octets 0 - 127)>
+    *
+    */
+    quotedPairRegex = /\\([ -~])/g,
+
+    /**
+     * @private
+     * Regex to match all the hexadecimal number inside encoded string
+     */
+    hexCharMatchRegex = /%([0-9A-Fa-f]{2})/g,
+
+    /**
+     * @private
+     * Regex to match non-latin characters
+     */
+    nonLatinCharMatchRegex = /[^\x20-\x7e\xa0-\xff]/g,
+
+    /**
+     * Replaces non-latin characters with '?'
+     *
+     * @private
+     * @param {string} val - The string which needs to be checked
+     */
+    getLatin = function (val) {
+        return String(val).replace(nonLatinCharMatchRegex, '?');
```

Comment:
`?` should be picked up from a constant.
Also, why is an explicit `String` call needed?